pkg/database/user.go

```diff
@@ -52,8 +52,7 @@ func (u *User) BeforeSave(tx *gorm.DB) error {
 	u.Email = project.TrimSpace(u.Email)
 	if u.Email == "" {
 		u.AddError("email", "cannot be blank")
-	}
-	if !strings.Contains(u.Email, "@") {
+	} else if !strings.Contains(u.Email, "@") {
 		u.AddError("email", "appears to be invalid")
```

Comment:
nit: `appears to be invalid` is weirdly passive. `invalid email address` or `email address must include @`?

Reply:
Done